templates/todo/projects/python-mongo/.repo/terraform/azure.yaml

```diff
@@ -15,5 +15,3 @@ services:
     project: ../../api/python
     language: py
     host: appservice
-infra:
```

Comment:
Did you intend this - it seems wrong?

Reply:
Oh - I see - It's also higher in this file :-)

Reply:
Somehow it got in there twice.  This remove the bottom one.